easybuild/easyconfigs/v/vispr/vispr-0.4.14-foss-2022a.eb

```diff
@@ -0,0 +1,40 @@
+# Author: Denis Kristak (Inuits)
+
+easyblock = 'PythonPackage'
+
+name = 'vispr'
+version = '0.4.14'
+
+homepage = 'https://pypi.org/project/vispr/'
+
+description = """
+VISPR - A visualization framework for CRISPR data.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+sources = ['%(name)s-%(version)s.tar.gz']
+patches = ['vispr_fix_pandas_version_change.patch']
```

Comment:
@deniskristak Please rename the patch file to also include the `vispr` version to which it applies: `vispr-0.4.14_fix_pandas_version_change.patch`